ui-admin/src/navbar/StudySidebar.tsx

```diff
@@ -120,6 +124,12 @@ export const StudySidebar = ({ study, portalList, portalShortcode }:
             className={sidebarNavLinkClasses} style={navStyleFunc}>Site Settings</NavLink>
         </li>
       </ul>}/>
+      <CollapsableMenu header={'Manage'} content={<ul className="list-unstyled">
```

Comment:
`Manage` sounds generic to me but I can't think of anything I'm super in love with... Maybe the section could be called `Study Staff` and the page is `Accounts`?

(As an aside, right now `Team members` has a different casing than the rest of the sidebar)